faker/providers/color/sk_SK/__init__.py

```diff
@@ -0,0 +1,11 @@
+from faker.providers import BaseProvider
+
+
+class Provider(BaseProvider):
```

Comment:
You must inherit from the standard color provider, not the base provider; otherwise, there will be some unexpected behavior. Please refer to #1279.

Reply:
Now, it should be OK.